changelogs/changelog.yaml

```
@@ -0,0 +1,85 @@
+---
+ancestor: null
+releases:
+  0.10.0:
+    changes:
+      bugfixes:
+      - k8s - Add exception handling when retrieving k8s client.
+      - k8s - Fix argspec for 'elements'.
+      - k8s - Use ``from_yaml`` filter with lookup examples in ``k8s`` module documentation
+        examples.
+      - k8s_service - Fix argspec.
+      - kubectl - Fix documentation in kubectl connection plugin.
+      major_changes:
+      - k8s_exec - New module for executing commands on pods via Kubernetes API.
+      - k8s_log - New module for retrieving pod logs.
+      minor_changes:
+      - k8s - Added ``persist_config`` option for persisting refreshed tokens.
+      security_fixes:
+      - kubectl - Warn about information disclosure when using options like ``kubectl_password``,
+        ``kubectl_extra_args``, and ``kubectl_token`` to pass data through to the
+        command line using the ``kubectl`` connection plugin.
+    fragments:
+    - 13-fix-elements-argspec.yaml
+    - 14-k8s_exec-new-module.yaml
+    - 16-k8s_log-new-module.yaml
+    - 33-k8s_service-fix-argspec.yaml
+    - 49-k8s-add-persist_config-option.yaml
+    - 51-kubectl-security-disclosure.yaml
+    - 52-kubectl-connection-docsfix.yaml
+    - 54-k8s-add-exception-handling.yaml
+    - 56-k8s-from_yaml-docs-examples.yaml
+    modules:
+    - description: Execute command in Pod
+      name: k8s_exec
+      namespace: .ansible.collections.ansible_collections.community.kubernetes.plugins.modules
+    - description: Fetch logs from Kubernetes resources
+      name: k8s_log
+      namespace: .ansible.collections.ansible_collections.community.kubernetes.plugins.modules
+    release_date: '2020-03-23'
+  0.11.0:
+    changes:
+      bugfixes:
+      - Make sure extra files are not included in built collection.
+      - Update GitHub Actions workflow for better CI stability.
+      - k8s_log - Module no longer attempts to parse log as JSON.
+      major_changes:
+      - helm - New module for managing Helm charts.
+      - helm_info - New module for retrieving Helm chart information.
+      - helm_repository - New module for managing Helm repositories.
+      minor_changes:
+      - Rename repository to ``community.kubernetes``.
+    fragments:
+    - 61-helm-new-modules.yaml
+    - 69-k8s_log-dont-parse-as-json.yaml
+    - 78-github-actions-workflow.yaml
+    - 81-rename-repository.yaml
+    - 85-exclude-unnecessary-files-when-building.yaml
+    modules:
+    - description: Manages Kubernetes packages with the Helm package manager
+      name: helm
+      namespace: .ansible.collections.ansible_collections.community.kubernetes.plugins.modules
```

```suggestion
      namespace: ''
```
Same for other similar entries in this changelog.

---

That seemed to work to patch over the issue; see comment in https://github.com/ansible-community/antsibull-changelog/issues/18.